movie_search/views.py

```diff
@@ -240,20 +241,82 @@ def search(request):
 
         return render(request, "media_search.html", context)
 
+def get_movie_genres(movie):
+    
+    movie_genres = []
 
-def movie_detail(request, obj_id):
-
-    movie_detail = media_api.get_media_data(f"/movie/{obj_id}")
-    # pprint("MOVIE DETAIL: ", movie_detail)
+    genres = movie.get("genres")
 
-    movie_videos = media_api.get_media_data(f"/movie/{obj_id}/videos")
+    if genres is not None:
+        for row in genres:
+            genre, inserted = Genre.objects.get_or_create(
+                name=row.get("name", ""),
+                genre_id=row.get("id", ""),
+            )
+            movie_genres.append(genre)
+    
+    return movie_genres
 
-    context = {
-        "movie_detail": movie_detail,
-        "movie_videos": movie_videos,
-        "type": "movie",
-    }
+def movie_detail(request, obj_id):
 
+    try:
+        movie_detail = Movie.objects.get(movie_id=obj_id)
+        context = {
```

Comment:
the context dict below has more items, do you need those here as well (in case of the success scenario)?